application/src/js/cms/_uk_countries_select.js

```diff
@@ -36,7 +36,7 @@ function ukCountriesSelect(element) {
       return;
     }
 
-    legend.parentElement.insertBefore(ukInputContainer, legend.nextSibling)
+    legend.parentElement.appendChild(ukInputContainer)
```

Comment:
Feels like the usefulness of having the "United Kindom" box fill in the rest of the boxes is diminished by putting it at the bottom.  Because you'll probably have ticked all four previous boxes as you go down the page before reaching the "United Kindom" one that woul have saved you three clicks.

Reply:
The reason we put it at the bottom is because twice the number of measures are at country rather than UK level. But we don't feel hugely strongly so feel free to move it to the top!

Reply:
@TheDoubleK I think @Nadiahuq’s reasoning is sound here. To be honest, the "UK" checkbox shortcut for selecting all hasn't turned out to be that useful, given that so far very few pages cover the whole UK. We could probably remove it.